<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1803</width>
    <height>1106</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_range_from">
    <property name="geometry">
     <rect>
      <x>1630</x>
      <y>650</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Niramit</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>From</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_step">
    <property name="geometry">
     <rect>
      <x>1620</x>
      <y>765</y>
      <width>160</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Niramit</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Step between points</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_graphicsView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>1531</width>
      <height>971</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_range_error">
    <property name="geometry">
     <rect>
      <x>1627</x>
      <y>760</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="leStepX">
    <property name="geometry">
     <rect>
      <x>1680</x>
      <y>805</y>
      <width>71</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Niramit</family>
     </font>
    </property>
    <property name="text">
     <string>20</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="leRangeTo">
    <property name="geometry">
     <rect>
      <x>1682</x>
      <y>690</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Niramit</family>
     </font>
    </property>
    <property name="text">
     <string>20</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_range">
    <property name="geometry">
     <rect>
      <x>1620</x>
      <y>610</y>
      <width>160</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Niramit</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Normalization range</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_range_to">
    <property name="geometry">
     <rect>
      <x>1630</x>
      <y>690</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Niramit</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>To</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="leRangeFrom">
    <property name="geometry">
     <rect>
      <x>1682</x>
      <y>650</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Niramit</family>
     </font>
    </property>
    <property name="text">
     <string>-20</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_open_file">
    <property name="geometry">
     <rect>
      <x>1660</x>
      <y>30</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Niramit</family>
      <pointsize>17</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Open File</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_scale">
    <property name="geometry">
     <rect>
      <x>1620</x>
      <y>110</y>
      <width>160</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Niramit</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Scale (in percents):</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="leStepY">
    <property name="geometry">
     <rect>
      <x>1680</x>
      <y>855</y>
      <width>71</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Niramit</family>
     </font>
    </property>
    <property name="text">
     <string>20</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_step_y">
    <property name="geometry">
     <rect>
      <x>1640</x>
      <y>861</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Niramit</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Y:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_step_x">
    <property name="geometry">
     <rect>
      <x>1640</x>
      <y>810</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Niramit</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="leScale">
    <property name="geometry">
     <rect>
      <x>1661</x>
      <y>150</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Niramit</family>
     </font>
    </property>
    <property name="text">
     <string>100</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="leMoveX">
    <property name="geometry">
     <rect>
      <x>1681</x>
      <y>280</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Niramit</family>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_move">
    <property name="geometry">
     <rect>
      <x>1609</x>
      <y>240</y>
      <width>160</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Niramit</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Move</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_move_x">
    <property name="geometry">
     <rect>
      <x>1610</x>
      <y>282</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Niramit</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>By X:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="leMoveY">
    <property name="geometry">
     <rect>
      <x>1681</x>
      <y>318</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Niramit</family>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_move_y">
    <property name="geometry">
     <rect>
      <x>1610</x>
      <y>320</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Niramit</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>By Y:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="leMoveZ">
    <property name="geometry">
     <rect>
      <x>1681</x>
      <y>358</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Niramit</family>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_move_z">
    <property name="geometry">
     <rect>
      <x>1610</x>
      <y>360</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Niramit</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>By Z:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="leRotateZ">
    <property name="geometry">
     <rect>
      <x>1681</x>
      <y>548</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Niramit</family>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="leRotateY">
    <property name="geometry">
     <rect>
      <x>1681</x>
      <y>508</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Niramit</family>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_rotate_z">
    <property name="geometry">
     <rect>
      <x>1610</x>
      <y>550</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Niramit</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>By Z:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="leRotateX">
    <property name="geometry">
     <rect>
      <x>1681</x>
      <y>470</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Niramit</family>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_rotate_x">
    <property name="geometry">
     <rect>
      <x>1610</x>
      <y>472</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Niramit</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>By X:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_rotate">
    <property name="geometry">
     <rect>
      <x>1609</x>
      <y>430</y>
      <width>160</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Niramit</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Rotate</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_rotate_y">
    <property name="geometry">
     <rect>
      <x>1610</x>
      <y>510</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Niramit</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>By Y:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_apply">
    <property name="geometry">
     <rect>
      <x>1640</x>
      <y>940</y>
      <width>130</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Niramit</family>
      <pointsize>17</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1803</width>
     <height>43</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
